src/design/components/atoms/RightLeftList.tsx

```diff
@@ -0,0 +1,64 @@
+import React, { useRef } from 'react'
+import { useEffectOnce } from 'react-use'
+import { focusFirstChildFromElement } from '../../lib/dom'
+import { useLeftToRightNavigationListener } from '../../lib/keyboard'
+
+interface LeftRightListProps {
+  className?: string
+  onBlur?: () => void
+  ignoreFocus?: boolean
+}
+const LeftRightList: React.FC<LeftRightListProps> = ({
```

Comment:
Component name seems opposite of file name `LeftRight` <-> `RIghtLeft`. Same name issue as above, wonder if there is a more intuitive name.

Reply:
It was a useless remnant, removed the component,
thanks!